scripts/rust/generate-openapi-bindings.sh

```diff
@@ -106,7 +106,7 @@ fi
 if [ -f "$RUST_FMT" ]; then
   cp "$RUST_FMT" "$tmpd"
   cp "$CARGO_LOCK" "$tmpd"
-  ( cd "$tmpd" && cargo fmt --all )
+  ( cd "$tmpd" && cargo fmt --all || true )
```

Comment:
Do we need this?

Reply:
Yes, seems in certain situations cargo fmt fails; this is not an issue because this code is autogenerated and not checked in.